src/mlpack/methods/ann/layer/constant_impl.hpp

```diff
@@ -30,6 +30,15 @@ Constant<InputDataType, OutputDataType>::Constant(
   constantOutput.fill(scalar);
 }
 
+template<typename InputDataType, typename OutputDataType>
+Constant<InputDataType, OutputDataType>::Constant(
+    const Constant& network) :
+    inSize(network.inSize),
+    outSize(network.outSize),
+    constantOutput(network.constantOutput)
+{
+}
```

Comment:
```suggestion
{
  // Nothing to do here.
}
```

Just for consistency.